File: athrill-account-server/Libraries/Model/Contents/ContentTypeModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Contents
{
    public class ContentTypeModel : BaseModel
    {
        public int ContentTypeId { get; set; }
        public string NameKey { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public override int Id => ContentTypeId;

        #region Link
        public  ICollection<ContentModel> Contents { get; set; }

        #endregion

    }
}

File: athrill-account-client/projects/at-models/src/lib/contents/content-type.model.ts
import { BaseModel } from '../at-base.model';
import { ContentModel } from './content.model';



export class ContentTypeModel extends BaseModel 
{
    public ContentTypeId: number;
    public NameKey: string;
    public DisplayName: string;
    public Description: string;
    public IsActive: boolean;

    public Id: number;

    public Contents : ContentModel[];
}